src/Entity/Field.php

```diff
@@ -90,7 +90,9 @@ public function __call(string $key = '', array $arguments = [])
         $value = $this->getTwigValue();
 
         if (is_array($value) && array_key_exists($key, $value)) {
-            return $value[$key];
+            $value = $value[$key];
+
+            return $value instanceof self ? $value->getTwigValue() : $value;
```

Comment:
This line could do with a comment, i think. I see what it does, but _why_ does it do this?